UI/UnoContoso/UnoContoso.Models/Data/Customer.cs

```diff
@@ -0,0 +1,78 @@
+//  ---------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+// 
+//  The MIT License (MIT)
```

Comment:
@jeromelaban : Are there implications for us legally ?

Reply:
We should add a 3rd-party notices file in Uno.Samples similar to https://github.com/unoplatform/uno/blob/master/THIRD-PARTY-NOTICES.md

Reply:
What should i do?

Reply:
Once I added the file. Please check if the contents of the file are correct. I didn't know how to add a subheader, so I added it to the bottom of the existing content. Can I do that?

Reply:
No. You only need to change THIRD-PARTY-NOTICES.md :)

![image](https://user-images.githubusercontent.com/33458222/97908286-7d14d200-1d14-11eb-8041-1b452acb93bf.png)

See https://github.com/unoplatform/Uno.Samples/pull/32#discussion_r516052648

Reply:
Thank you. Fixed